internal/oci/sandbox.go

```diff
@@ -52,3 +52,18 @@ func GetSandboxTypeAndID(specAnnotations map[string]string) (KubernetesContainer
 	}
 	return ct, id, nil
 }
+
+// SandboxAnnotationsPassThrough passes through the annotations specified by 'vals' from the sandboxes set of
+// annotations through to every container in the pod. Kubernetes only passes metadata annotations to the
+// RunPodSandbox request, so annotations that are meant to be available for use/checking for individual
+// containers need some way to know they were passed for the pod.
+func SandboxAnnotationsPassThrough(podAnnots map[string]string, containerAnnots map[string]string, vals ...string) {
```

Comment:
Nit: Could combine the map type for `podAnnots` & `containerAnnots` like `podAnnots, containerAnnots map[string]string), vals ...string`

Reply:
Good point!